proto/stream_plan.proto

```diff
@@ -129,6 +133,12 @@ message BarrierMutation {
     ThrottleMutation throttle = 10;
     // Drop subscription on mv
     DropSubscriptionsMutation drop_subscriptions = 12;
+    // Start backfilling for specific fragments
+    // This is separated from `ThrottleMutation::NO_RATE_LIMIT`.
+    // This is because user may concurrently rate limit + use backfill order control.
+    // If we use rate limit to pause / resume backfill fragments, if user manually
+    // resumes some fragments, this will overwrite the backfill order configuration.
```

Comment:
I think this is not bad, if the user intends some intervention? As a result, we don't have to introduce a new control plane.

Reply:
We can introduce separate path / features for that, rather than overlap into rate limit.